### PR TITLE
OPAL-625 Update README

### PR DESCRIPTION
**Pull request by AM-USAF**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+### Pytest
+<p>It is highly recommended that each pytest name is descriptive of what it is testing. It is also recommended to add a short docstring at the beginning of the function describing what the test does to help with debugging and clarity.</p>
+
+### Pylint
+<p>If you disable a pylint rule, you must include a comment explaining why you disabled that pylint rule.</p>
+
+### Other/Misc
+<p>Verify that every sentence, documentation, and comment is coherent, comprehensible, and adheres to proper grammar.</p>
+
+<p>PEP-8 <a href="https://peps.python.org/pep-0008/#indentation">Indentation</a>.</p>
```
Comment (JR-USAF) [inline CONTRIBUTING.md:195]: What this contribution guide taken from Weave?

File: README.md [hunk 2]
```diff
@@ -169,3 +137,45 @@ Run the following command to output help from CLI
...
+
+### SQL Connection
+An external postgreSQL service is required for the tests to run and the following environment variables must be set.
+- `SQL_HOST`
+- `SQL_PASSWORD`
+- `SQL_PORT`
+- `SQL_USERNAME`
+
+### Running Tests in AWS
+
```
Comment (JR-USAF) [inline README.md:156]: Is this only needed for Argo tests?

File: CONTRIBUTING.md (hunk 1 above)
Comment (AM-USAF) [reply CONTRIBUTING.md:195]: Yes, I tried to remove all instances of weave in it and edit it so it is applicable to our repo

File: README.md (hunk 2 above)
Comment (AM-USAF) [reply README.md:156]: The running test in AWS part should run all pytests not only argo tests but I haven't actually tested it, it's from the original readme. Is that what you mean?

Comment (JR-USAF) [reply README.md:156]: Maybe we should say something above this step saying something to affect of. All tests besides argo related tests can be run without additional setup. Continue on for information on running argo tests. Maybe also explain the `RUN_ARGO_TESTS` flag.

Comment (AM-USAF) [reply README.md:162]: I've updated the readme

**Review by JR-USAF — APPROVED** [review]

**Review by JP-USAF — APPROVED** [review]

**Pull request merged**